docs/backup-restore.md

```diff
@@ -1,134 +1,208 @@
 
-### MySQL (restore)
+{% capture notice--warning %}
+**Note**
 
-If you use MySQL and `mysqldump` to create a backup file, use `mysql` command to restore the file as specified [in the MySQL doc](https://dev.mysql.com/doc/mysql-backup-excerpt/8.0/en/reloading-sql-format-dumps.html).
+The backup methods by database listed below are just examples of some of the databases that ScalarDB supports.
+{% endcapture %}
 
-### PostgreSQL (restore)
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-If you use PostgreSQL and `pg_dump` to create a backup file, use `psql` command to restore the file as specified in [the PostgreSQL doc](https://www.postgresql.org/docs/current/backup-dump.html#BACKUP-DUMP-RESTORE).
+<div id="tabset-3">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Amazon_RDS_or_Azure_Database_for_MySQL_or_PostgreSQL3', 'tabset-3')" id="defaultOpen-3">Amazon RDS or Azure Database for MySQL or PostgreSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'Cassandra3', 'tabset-3')">Cassandra</button>
+  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL3', 'tabset-3')">Cosmos DB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB3', 'tabset-3')">DynamoDB</button>
+  <button class="tablinks" onclick="openTab(event, 'MySQL3', 'tabset-3')">MySQL</button>
+  <button class="tablinks" onclick="openTab(event, 'PostgreSQL3', 'tabset-3')">PostgreSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'SQLite3', 'tabset-3')">SQLite</button>
+</div>
 
-### SQLite (restore)
+<div id="Amazon_RDS_or_Azure_Database_for_MySQL_or_PostgreSQL3" class="tabcontent" markdown="1">
 
-Use [`.restore` command](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_).
+You can restore to any point within the backup retention period by using the automated backup feature.
+</div>
+<div id="Cassandra3" class="tabcontent" markdown="1">
 
-### Amazon RDS or Azure Database for MySQL/PostgreSQL (restore)
+First, stop all the nodes of the Cassandra cluster. Then, clean the `data`, `commitlog`, and `hints` directories, and place the backups (snapshots) in each node.
 
-You can restore to any point within the backup retention period with the automated backup feature.
+After placing the backups (snapshots) in each node, start all the nodes of the Cassandra Cluster.
+</div>
+<div id="Cosmos_DB_for_NoSQL3" class="tabcontent" markdown="1">
 
-### Cassandra (restore)
+Follow the official Azure documentation for [restore an account by using Azure portal](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal). After restoring a backup, change the default consistencies of the restored databases to `STRONG`. In addition, you should use the mid-time of the paused duration as the restore point as previously explained.
```

Comment:
```suggestion
Follow the official Azure documentation for [restore an account by using Azure portal](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal). After restoring a backup, [configure the default consistency level](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-manage-consistency#configure-the-default-consistency-level) of the restored databases to `STRONG`. In addition, you should use the mid-time of the paused duration as the restore point as previously explained.
```
This is a minor thing, but the `default consistency level` are proper word in the official document. So, I think it would be better to use those words instead of `default consistencies`.

Also, I think it would be better to add an official document link to configure the default consistency level of Cosmos DB.

Reply:
I see. I agree, especially with providing a link👍